src/v3/test/integration/authenticator-enroll-security-question.test.tsx

```diff
@@ -255,8 +255,8 @@ describe('authenticator-enroll-security-question', () => {
         .toEqual('The security question answer must be at least 4 characters in length');
 
       // switch to predefined question form
-      await user.click(await findByLabelText(/Choose a security question/));
-      await findByLabelText('Choose a security question', { selector: 'select' });
```

Comment:
Should we add an extra set of actions that also does some typing and then asserts that the first one is a certain question?